src/DDTrace/Integrations/IntegrationsLoader.php

```diff
@@ -93,6 +94,8 @@ public function __construct(array $integrations)
                 '\DDTrace\Integrations\Memcached\MemcachedSandboxedIntegration';
             $this->integrations[MysqliSandboxedIntegration::NAME] =
                 '\DDTrace\Integrations\Mysqli\MysqliSandboxedIntegration';
+            $this->integrations[MongoSandboxedIntegration::NAME] =
+                '\DDTrace\Integrations\Mongo\MongoSandboxedIntegration';
```

Comment:
Nit: can you move this before Mysqli so it's in alphabetical order?

Reply:
Also, I should probably learn the alphabet :)